README.md

```diff
@@ -1,7 +1,9 @@
 ## SDG Repositiry
 
-This is the collaborative space for our tools and data around the SDG indicators. 
+[![goodtables.io](https://goodtables.io/badge/github/roll/SDG.svg)](https://goodtables.io/github/roll/SDG)
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/8

On the bottom of the page, we can find the code to add the goodtables.io (`data: valid/invalid`) badge. So, on every repo change, we can see the validation status in README.